Add answer to 8.25 as well as some formatting and a gitignore.

natesholland opened the pull request:
I had to add some extra stuff to get this to compile.
But I have checked this and I know that all the latex is valid.
See https://proofwiki.org/wiki/Graph_is_Bipartite_iff_No_Odd_Cycles for more
information.

I also added a gitignore to ignore the pdf and log and other artifacts of
compiling .tex files.

natesholland committed:
Add partial answer to 8.25 and a gitignore.

See https://proofwiki.org/wiki/Graph_is_Bipartite_iff_No_Odd_Cycles for more
information.

I also added a gitignore to ignore the pdf and log and other artifacts of
compiling .tex files.

ryandougherty commented:
This does not conform to the style of other exercises, and the main .tex file is the one that is used to generate the pdf.

ryandougherty commented:
And plus this solution to 8.25 only covers the first half of the question.

natesholland commented:
Ok I fixed the formatting and added a note about being partially complete, then squashed it down into one commit. I can come back later when I have time and sketch out the other direction of the proof.

natesholland committed:
Add compiling instructions to the README.

This caught me up for a bit so I added instructions to the README.

natesholland commented:
As I typed up the question I realized that this also asks us to prove that bipartite is also in NL. To be completely honest I don't quite know how to do that. I have a good sketch in my head of how to show that no odd graphs implies bipartite but I'm really weak on proofs about space complexity.
